stubs/django/template/__init__.pyi

```diff
@@ -8,10 +8,5 @@ from typing import Any, Mapping
 class TemplateSyntaxError(Exception):
     pass
 
-class Template: ...
```

Comment:
You've correctly identified that `Template` and `Engine` are not actually declared in `__init__.py`. There were in this file, however, because they are usually imported as:
```
from django.template import Template
```
Rather than:
```
from django.template.base import Template
```
This is possible because of the import statement in the `__init__.py` file:
https://github.com/django/django/blob/1cba3207867fb168788ad6120770d0398b6b2807/django/template/__init__.py#L61

So that's all a long winded way of saying you're right to move `Template` and `Engine` to their proper locations, but you also need to add some imports to this `__init__.py`:
```
from .base import Template
from .engine import Engine
```

Reply:
Makes sense added.